members/B000575.yaml

```diff
@@ -4,45 +4,53 @@ contact_form:
   action: ""
   steps:
     - visit: "https://www.blunt.senate.gov/public/index.cfm/contact-roy"
+    - wait:
+      - value: 1
+    - find:
+      - selector: div.js-form-toggle.feedback-button a
```

Comment:
Are you sure this is a unique selector?

Reply:
yeahh, I'm not loving it but it's definitely unique 

[![Screenshot from Gyazo](https://gyazo.com/66ec1b23125b9102d82bd8906a4d27ee/raw)](https://gyazo.com/66ec1b23125b9102d82bd8906a4d27ee)